src/components/TodoList/TodoList.tsx

```diff
@@ -1,210 +1,82 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { memo } from 'react';
+import { Todo } from '../../types/Todo';
+import { useAppDispatch } from '../../app/hooks';
+import { actions } from '../../features/currentTodo';
+
+interface Props {
+  todos: Todo[];
+  selectedTodoId: number;
+}
+
+export const TodoList: React.FC<Props> = memo(({
+  todos,
+  selectedTodoId,
+}) => {
```

Comment:
again. You dont need to use props passing

redux state is avalible from any component

just take todos and currentTodo from global state